docs/configure-custom-values-envoy.md

```diff
@@ -70,11 +64,11 @@ You can configure them using the same syntax as the requests and limits of Kuber
 envoy:
   resources:
     requests:
-      cpu: 200m
-      memory: 256Mi
+      cpu: 1000m
+      memory: 2Gi
     limits:
-      cpu: 300m
-      memory: 328Mi
+      cpu: 2000m
+      memory: 4Gi
```

Comment:
The previous values are based on the very old test result.
So, I updated it to somewhat large (somewhat safe) values.
I will update these values based on our new testing in the production-simulated environment in the future.
Also, I will set the default `limits` configuration in the future.